src/features/TakeNow/ui/MultiInformantBadge.tsx

```diff
@@ -0,0 +1,36 @@
+import { useMultiInformantState } from '~/entities/applet/model/hooks';
+import { MultiInformantBadgeTile } from '~/features/TakeNow/ui/MultiInformantBadgeTile';
+import { Theme } from '~/shared/constants';
+import { Box } from '~/shared/ui';
+import { useLaunchDarkly } from '~/shared/utils/hooks/useLaunchDarkly';
+
+export const MultiInformantBadge = () => {
+  const { getMultiInformantState, isInMultiInformantFlow } = useMultiInformantState();
+  const { flags: featureFlags } = useLaunchDarkly();
+
+  if (!isInMultiInformantFlow() || !featureFlags.enableMultiInformant) {
+    return null;
+  }
+
+  const { sourceSubject, targetSubject } = getMultiInformantState();
+
+  if (!sourceSubject || !targetSubject) {
+    return null;
+  }
+
+  return (
+    <Box
+      display="flex"
+      padding="4px"
```

Comment:
🦅 Although Figma says 4px padding, it's also got an "inner-aligned" border, which is a Figma-specific trick that doesn't actually map directly to CSS. In Figma, the 1px border is taking up 1px of the 4px padding, leaving 3px padding left:
```suggestion
      padding="3px"
```

Reply:
Incredible. Thanks for the tip, I'll keep an eye out for this in the future